lib/core/src/msParam.cpp

```diff
@@ -570,6 +570,13 @@ int clearMsParam(msParam_t* msParam, int freeInOutStruct)
                 clearExecCmdOut(msParam->inOutStruct);
                 std::free(msParam->inOutStruct);
             }
+            // Do not free KeyValPair_MS_T unconditionally as this will lead
+            // to a use-after-free. Additionally, this behavior matches the default
```

Comment:
Feels like we're missing a word after `... to a use-after-free.`.

Reply:
Jokingly: `... this well lead to a contretemps; a use-after-free error.`

I feel like `use-after-free` is 'noun enough' to end on, but perhaps ending in `error` or `memory error` may help?

Reply:
Appending `error` is good enough.

Think about future readers of the code. Try not to leave room for confusion.

Reply:
`error` added.